packages/11ty/_includes/components/objects-data.webc

```diff
@@ -0,0 +1,2 @@
+<script data-objects webc:keep type="application/json" @raw="JSON.stringify(objects)">
+</script>
```

Comment:
Is the following to verbose?
```suggestion
<script data-objects webc:keep type="application/json" @raw="objectsJSON"></script>

<script webc:setup>
/**
 * Write the publication objects data to JSON on the page,
 * making it available via `querySelector('script[data-object]')`
 */
const objectsJSON = JSON.stringify(objects)
</script>
```

Reply:
The comment is good context, but we can't pass the `objectsJSON` variable between the two `<script>` blocks -- if we could, we wouldn't need to write the JSON at all